srcalc/src/test/java/gov/va/med/srcalc/service/MockAdminService.java

```diff
@@ -0,0 +1,63 @@
+package gov.va.med.srcalc.service;
+
+import gov.va.med.srcalc.domain.model.*;
+
+import java.util.List;
+import java.util.Set;
+
+/*
+ * Created to support the EditRiskModel which needs to call getAllSpecialties()
+ * Implement other methods as needed.
+ * 
+ */
+public class MockAdminService extends MockModelService implements AdminService  {
+
+    public MockAdminService() 
+    {        
+    }
+    
+    @Override
+    public List<AbstractVariable> getAllVariables() {
+        return null;
+    }
+
+    @Override
+    public AbstractVariable getVariable(String key)
+            throws InvalidIdentifierException {
+        return null;
```

Comment:
What's the point of returning null for these? I would also put javadoc comments on the unimplemented methods to let other developers know that they aren't doing anything.